packages/client/src/styles/variables/styleConstants.ts

```diff
@@ -0,0 +1,15 @@
+import { css } from 'styled-components';
+
+export const DEPTH_CONTAINER = '10px 10px 10px rgba(0, 0, 0, 0.25)';
+
+export const SIZE_BORDER_RADIUS = '20px';
+
+export const SIZE_BORDER_WIDTH_TABLE = '2px';
```

Comment:
```suggestion
export const BORDER_RADIUS_SIZE = '20px';

export const BORDER_WIDTH_TABLE = '2px';
```
https://github.com/FancyCodersYAP/UNO-fancy/pull/10#discussion_r1126043512